apps/cms/src/@types/Order.ts

```diff
@@ -4,14 +4,13 @@
 
 export class Order {
   constructor(
-    public order_id = '',
-    public date = new Date(),
-    public order_person = '',
-    public image_url = '',
-    public item = '',
-    public qty = 0,
-    public size = '',
-    public colour = ''
-
+    public id = '',
+    public transaction_id = '',
+    public transaction_time = '',
+    public payment_method = '',
+    public customerEmail = '',
+    public status = '',
+    public updatedAt = '',
+    public createdAt = ''
```

Comment:
Is there any reason some fields are `snake_case` and some fields are `camelCase`?

Reply:
No idea, it was already inconsistent prior to this PR: https://github.com/ntuscse/website/pull/153

Reply:
In that case, can you help updating it to be consistent? Thank you!